src/components/header/header-constants.tsx

```diff
@@ -0,0 +1,18 @@
+export const enum HeaderEnum {
```

Comment:
I think we have a area we should put enums

Reply:
Which file in the project could you point me towards?

Reply:
Follow this

![image](https://github.com/OpenBCca/openbc-web/assets/40151580/2703b405-6461-426a-9b45-785d71ed6258)


You should create a separate file for enum. Don't use a single file for multiple enum.

Reply:
Say, header-map in utils/maps, header-enum in dataModels/enum header-data in dataModels/dictionaries?

Reply:
I think it makes more sense:

1) Enum folder at the src level
2) header-data is in the same folder as header component
3) utils/header-map/header-map